Twig/TwigYuiExtension.php

```diff
@@ -90,6 +90,14 @@ public function yuiConfigLoaderFunction($configObject = '')
             'modules' => [],
         ];
 
+        $combine = $this->configResolver->getParameter('yui.combine', 'ez_platformui');
+
+        if (true === $combine) {
```

Comment:
Ok, I'll change it. Consistency is important.

But I still love it and I think it's better :-)